### ISSUE TITLE
question: What's the difference between change start page types?

### ISSUE THREAD

**Issue by s7san007** [inotia00/ReVanced_Extended#2551]
### Application

YouTube

### Question description

I can't see any difference between "once" and "always".
I've restarted the app once, twice, many times and I see no difference.

### Acknowledgements

- [X] This issue is not a duplicate of an existing question.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] I have written the title and contents in English.

**Comment by inotia00** [comment]
For some start page types, the start page is changed only once on a cold start
(e.g. Library)

This setting makes the start page change even on a warm start

https://github.com/inotia00/ReVanced_Extended/issues/2395#issuecomment-2368848355
